cloud/passportauth/README.md

```diff
@@ -0,0 +1,75 @@
+# RainCatcher Passport Auth
+Raincatcher authentication and authorization module using [PassportJS][1]. This module creates and initializes a Passport authentication service using Passport's local strategy.
+This allows express routes to be protected from requests by checking if the user is authenticated and/or authorized with the required role. This module also
+uses persistent login sessions using [express-session][2].
+
+
+## Quick Start
+#### Setup & Pre-requisites
+1. **Implement BaseUser Interface**
+
+    Implement the BaseUser interface defined in `./src/user/BaseUser.ts` in order to allow for retrieving the required user data from your data source.
+    See `./example/UserApi.ts` for a sample implementation of this interface
+
+2. **Initialize Passport auth service**
+   ```typescript
+       ...
```

Comment:
use the triple bactick but not the four spaces, this makes the code indented by 4 spaces.